api/src/services/import-export/get-import-worker.ts

```diff
@@ -0,0 +1,19 @@
+import Tinypool from 'tinypool';
+import { createRequire } from 'node:module';
+
+let worker: Tinypool | undefined;
```

Comment:
@paescuj why do we need to keep this object in memory?
I believe the average use cases is to do one or two imports but that's not a frequent thing.
So, I think we can create it every time we want to make an import so we don't fill memory with unused stuff 🤔

Reply:
@joselcvarela This is to ensure the same pool is (re-)used for all import tasks, otherwise `tinypool` cannot reliably manage the threads. I don't think the object will consume notable amount of memory.
Note that the thread itself will be cleaned-up by `tinypool` after the task has finished, this wasn't the case without `tinypool`.